pkgs/development/ocaml-modules/gluten/eio.nix

```diff
@@ -0,0 +1,12 @@
+{ lib, buildDunePackage, gluten, eio }:
+
+buildDunePackage {
+  pname = "gluten-eio";
+  inherit (gluten) src version;
+
+  propagatedBuildInputs = [ gluten eio ];
+
+  meta = gluten.meta // {
+    description = "EIO runtime for gluten";
+  };
+}
```

Comment:
Unsure about this one, I don't know if such meta-packages are normally included in the nixpkgs tree.

Reply:
what does "meta-package" mean here?

Reply:
Using it in the same sense as:

> The difference between a meta package and a regular package is that a meta package is empty and exists purely to link related packages together via dependencies. A meta package, often (though not always) titled with the "-meta" suffix, provides similar functionality to a package group in that it enables multiple related packages to be installed or uninstalled simultaneously. 

https://wiki.archlinux.org/title/Meta_package_and_package_group

> One of the handy features of apt (the packaging system used by Ubuntu) is the use of metapackages. These packages do not contain actual software, they simply depend on other packages to be installed. This setup allows entire sets of software to be installed by selecting only the appropriate metapackage. For example, an Ubuntu user can install the Kubuntu environment (KDE and all its associated programs) by selecting "kubuntu-desktop." 

https://help.ubuntu.com/community/MetaPackages

Reply:
Then `gluten-eio` is not a meta package. Here's its source code https://github.com/anmonteiro/gluten/tree/master/eio

Reply:
Oh, apologies misread the change a bit and thought it was only for the propagatedBuildInputs